pygmt/src/grdview.py

```diff
@@ -18,18 +18,19 @@
 @use_alias(
     C="cmap",
     G="drapegrid",
-    N="plane",
     Q="surftype",
     Wc="contourpen",
     Wm="meshpen",
-    Wf="facadepen",
     I="shading",
     f="coltypes",
     n="interpolation",
 )
 def grdview(  # noqa: PLR0913
     self,
     grid: PathLike | xr.DataArray,
+    plane: float | bool = False,
```

Comment:
Following our naming convention, the variables `facadepen` and `facadefill` should be `facade_pen`/`facade_fill` instead, but we also have `contourpen` and `meshpen`, so maybe we should keep `facadepen`/`facadefill` here.

Reply:
I've reopened https://github.com/GenericMappingTools/pygmt/issues/2014 for more discussions.